### ISSUE TITLE
Allow `vts` build without `matlab` implementation

### ISSUE THREAD

**Issue by scottprahl** [VirtualPhotonics/VTS#140]
Currently executing 

    ./BuildTestRelease.ps1 

requires a `matlab` installation.  Perhaps the `matlab` builds could be made optional for those who just want to use `vts` in a `JuptyerLab` environment?

**Comment by hayakawa16** [comment]
*Like* this suggestion!  We are currently working on python scripts so this comment is right in line with our next steps.  We will work on a way to opt in and out of these additional tools.

**Comment by hayakawa16** [comment]
I have determined command to check if matlab is installed or not.  I thought I could just edit BuildTestRelease and check before command  to Invoke BuildTestReleaseMATLAB, however matlab is also invoked in BuildTestReleaseMCCL.  So I added the same if check before that call.  I have tried updated code on:
1) windows with matlab installed
2) linux with matlab installed
3) linux with no matlab available
and these run as they should.
I have pushed these changes to the branch associated with this issue and will create a PR for review.
